src/main/java/donnafin/model/person/Liability.java

```diff
@@ -30,18 +55,34 @@ public static boolean isValidLiability(String test) {
 
     @Override
     public String toString() {
-        return value;
+        return "Liability{" +
+                "name='" + name + '\'' +
+                ", type='" + type + '\'' +
+                ", value='" + value + '\'' +
+                ", remarks='" + remarks + '\'' +
+                '}';
```

Comment:
I'm thinking perhaps for Asset, Liability, Policy we should have `toShortString()` as well that returns smth like "Policy Name (Policy Type)" for use in the table and command output and so on. Wdyt?